ioctl/newcmd/alias/aliasremove_test.go

```diff
@@ -13,37 +16,39 @@ import (
 )
 
 func TestNewAliasRemoveCmd(t *testing.T) {
+	require := require.New(t)
 	// mock a client
 	ctrl := gomock.NewController(t)
 	client := mock_ioctlclient.NewMockClient(ctrl)
-	client.EXPECT().SelectTranslation(gomock.Any()).Return("%s is removed", config.English).Times(6)
-
-	// configuration files to temporary files
-	defaultConfigFile := config.DefaultConfigFile
-	config.DefaultConfigFile = filepath.Join(os.TempDir(), "config.default")
-	defer func() {
-		_ = os.Remove(config.DefaultConfigFile)
-		config.DefaultConfigFile = defaultConfigFile
-	}()
-
 	cfg := config.Config{
 		Aliases: map[string]string{
 			"a": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx",
-			"b": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx",
+			"b": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542se",
 			"c": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542s1",
 		},
 	}
-	client.EXPECT().Config().Return(cfg).AnyTimes()
-	cmd := NewAliasRemove(client)
-	res, err := util.ExecuteCmd(cmd, "a")
-	require.NotNil(t, res)
-	require.NoError(t, err)
+	client.EXPECT().AliasMap().Return(map[string]string{
+		"a": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx",
+		"b": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542se",
+		"c": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542s1",
+	}).Times(2)
+	client.EXPECT().Config().Return(cfg).Times(2)
+
+	t.Run("remove alias", func(t *testing.T) {
+		client.EXPECT().SelectTranslation(gomock.Any()).Return("%s is removed", config.English).Times(5)
+		client.EXPECT().DeleteAlias("io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx").Return(nil)
```

Comment:
the argument of DeleteAlias is alias name

Reply:
Seems `DeleteAlias` received address (value) instead of alias (key) as argument, see below:
https://github.com/iotexproject/iotex-core/blob/f53aec1499af1dac8893ce88e3fc4611b38be6ef/ioctl/newcmd/alias/aliasremove.go#L60